contributors.json

```
@@ -304,6 +304,10 @@
     "linkedin": "www.linkedin.com/in/shubham-agarwal16",
     "name": "Shubham Agarwal"
   },
+  "Shubham-Kumar-Baranwal": {
```

```suggestion
  "baranwal": {
```

The key must be Github user id